src/main/java/com/google/api/generator/gapic/composer/samplecode/ServiceClientSampleCodeComposer.java

```diff
@@ -842,6 +849,131 @@ public static String composeStreamCallableMethodHeaderSampleCode(
 
     return bodyStatements;
   }
+
+  private static List<Statement> composeStreamClientSampleCodeBodyStatements(
+      Method method, VariableExpr clientVarExpr, AssignmentExpr requestAssignmentExpr) {
+    List<Expr> bodyExprs = new ArrayList<>();
+
+    // Create responseObserver variable expression.
+    // e.g. ApiStream<EchoResponse> responseObserver
+    TypeNode responseObserverType =
+        TypeNode.withReference(
+            ConcreteReference.builder()
+                .setClazz(ApiStreamObserver.class)
+                .setGenerics(method.inputType().reference())
+                .build());
+    VariableExpr responseObserverVarExpr =
+        VariableExpr.withVariable(
+            Variable.builder().setName("responseObserver").setType(responseObserverType).build());
+    // Create Anonymous Class contains methods: onNext, onError, onCompleted.
```

Comment:
Nit: Grammar and regular casing rules apply in comments. "Create anonymous class" or "Create an anonymous class for ApiStreamObserver that contains the methods onNext, onError, and onCompleted."

Reply:
Agree. Thanks for correcting.